### DISCUSSION TITLE
warning: unused parameter 'allow_exceptions'

### DISCUSSION THREAD

**Discussion by ghost** [nlohmann/json#2298] [General]
> clang++ -Wall -Wextra -pedantic -std=c++20 -g -fsanitize=address main.cpp ppm.cpp -o tracer
> In file included from main.cpp:22:
> ./json.hpp:23786:44: warning: unused parameter 'allow_exceptions' [-Wunused-parameter]
>                                 const bool allow_exceptions = true,
>                                            ^
> 1 warning generated.

Perhaps a `(void)allow_exceptions;` would fix this.

**Comment by nlohmann** [comment]
Good catch! I think this is a bug. The parameter should actually be used. I encountered this yesterday while working on #2297. As there are still problems on that PR, I will create a PR just for this issue.

**Reply by nlohmann** [reply]
Related PR: #2299.

**Reply by nlohmann** [reply]
Merged.
